src/shared/utmp-wtmp.h

```diff
@@ -24,6 +26,15 @@ int utmp_wall(
         bool (*match_tty)(const char *tty, void *userdata),
         void *userdata);
 
+static inline bool setutxentp(void) {
```

Comment:
hmm, the "p" suffix was supposed to indicate that extra pointer level of indirection the destructors get, i.e. free() takes a pointer, but freep() gets a pointer to a pointer...

Doesn't matter much, but this naming logic doesn't really make sense in this case...